cristin-import/src/test/java/no/unit/nva/cristin/mapper/CristinMapperTest.java

```diff
@@ -401,23 +400,59 @@ public void mapThrowsMissingFieldsExceptionWhenNonIgnoredFieldIsMissing() {
         mapSetsNameToNullWhenBothFamilyNameAndGivenNameAreMissing();
     }
 
+    @ParameterizedTest(name = "map returns journal with DOI URI when input is Journal with DOI:{0}")
+    @CsvSource({
+        "http://dx.doi.org/10.5750/ejpch.v5i1.1209, https://doi.org/10.5750/ejpch.v5i1.1209",
+        "https://doi.org/10.1016/j.toxlet.2020.09.006, https://doi.org/10.1016/j.toxlet.2020.09.006",
+        "DOI:10.1080/02699052.2017.1312145, https://doi.org/10.1080/02699052.2017.1312145",
+        "doi:10.1080/02699052.2017.1312145, https://doi.org/10.1080/02699052.2017.1312145",
+        "10.1016/j.toxlet.2020.09.006, https://doi.org/10.1016/j.toxlet.2020.09.006"
+    })
+    public void mapReturnsJournalWithDoiWhenInputIsJournalResultWithDoi(String inputDoi, String expectedDoiUri) {
+        CristinObject cristinObject =
+            CristinDataGenerator.randomObject(CristinSecondaryCategory.JOURNAL_ARTICLE.toString());
+        cristinObject.getJournalPublication().setDoi(inputDoi);
+        Publication publication = cristinObject.toPublication();
+        assertThat(publication.getEntityDescription().getReference().getDoi(), is(equalTo(URI.create(expectedDoiUri))));
+    }
+
+    @Test
```

Comment:
Test for validating that the test configuration that disables online validation is loaded